src/index.html

```diff
@@ -5,8 +5,468 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <img class="header__img" src ="images/logo.png"
+          alt="logo"href="#home">
+        <nav class="nav">
+          <ul class="nav__list"></ul>
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#"
+            >
+              Video
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops &#38 computers
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link" href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="is-active" href="#"
+            >
+              Apple
+            </a>
+          </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="container">
+          <div class="product-card" data-qa="card">
+            <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K   Monoblock"
+            class="product-card__image"
+            >
```

Comment:
supposed to be 2 spaces for attributes
```suggestion
            <img
              src="./images/imac.jpeg"
              alt="APPLE A1419 iMac 27 Retina 5K   Monoblock"
              class="product-card__image"
            >
```